docs/patterns/notification.md

```diff
@@ -19,6 +19,12 @@ The default variant should be used to display global messages.
 View example of the default notification pattern
 </a>
 
+<div class="p-notification--caution">
+  <p class="p-notification__response">
+    <span class="p-notification__status">Warning:</span>This pattern requires JS to be functional.
+  </p>
+</div>
```

Comment:
Could you move this to the notification pattern that requires JS. I assume it's the one that can be hidden.

Reply:
Updated 👍